$OpenBSD: patch-gfx_skia_skia_src_core_SkCpu_cpp,v 1.1 2019/10/08 20:16:08 sthen Exp $

Firefox generates its own sys/auxv.h header in build-aarch64/dist/system_wrappers
which conflicts with skia's method of detecting whether the OS has sys/auxv.h
itself. This is used only to detect cpu features on arm64 (falling back to not
using those features) so seems safe to ignore.

https://bugzilla.mozilla.org/show_bug.cgi?id=1587116

Index: gfx/skia/skia/src/core/SkCpu.cpp
--- gfx/skia/skia/src/core/SkCpu.cpp.orig
+++ gfx/skia/skia/src/core/SkCpu.cpp
@@ -70,7 +70,7 @@
         return features;
     }
 
-#elif defined(SK_CPU_ARM64) && __has_include(<sys/auxv.h>)
+#elif !defined(__OpenBSD__) && defined(SK_CPU_ARM64) && __has_include(<sys/auxv.h>)
     #include <sys/auxv.h>
 
     static uint32_t read_cpu_features() {
